visual-js/.changeset/wet-pianos-work.md

```diff
@@ -0,0 +1,6 @@
+---
+"@saucelabs/visual": minor
+"@saucelabs/nightwatch-sauce-visual-service": minor
```

Comment:
Are we sure these should be 'minor'? In JS ecosystem `minor` bumps in a prerelease version (`0.x.x`) would not be considered 'compatible'